src/main/java/com/example/demo/controller/ItemController.java

```diff
@@ -84,14 +83,13 @@ public String detail(@PathVariable("id") Integer id, Model model) {
 		model.addAttribute("logo", "AyaDesign");
 
 		Optional<Item> itemSearch = itemService.findById(id);
-		if (itemSearch.isPresent()) {
+		try {
 			Item item = itemSearch.get();
```

Comment:
.getでNoSuchElementExceptionを起こしてキャッチではなく以下のような書き方で書くほうがよいです。
```suggestion
			Item item = itemSearch.orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND));
```

Reply:
他にも類似の箇所あるとおもうので横展開お願いします！

Reply:
OKです！